receipt_agent/receipt_agent/agents/label_suggestion/tools/label_suggestion_tools.py

```
@@ -8,13 +8,15 @@
 import logging
 import time
 from dataclasses import dataclass
+from datetime import datetime, timezone
 from typing import Any, Callable, Optional
```

Ruff (0.14.10)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)